bentoml/server/open_api.py

```diff
@@ -61,9 +61,34 @@ def get_open_api_spec_json(bento_service):
                 "in JSON format and must contain a `request_id` field, which can be "
                 "obtained from any BentoService API response header",
                 responses=default_response,
-            )
+            ),
+            post=OrderedDict(
+                tags=["infra"],
+                description="Predictions feedback endpoint. Expecting feedback request "
+                "in JSON format and must contain a `request_id` field, which can be "
+                "obtained from any BentoService API response header",
+                requestBody=OrderedDict(
+                    required=True,
+                    content={
+                        "application/json": {
+                            "schema": {
+                                "type": "object",
+                                "required": ["request_id", "feedback"],
+                                "properties": {
+                                    "request_id": {"type": "uuid"},
+                                    "feedback": {"type": "string"},
```

Comment:
We don't want to confuse the user that feedback is a field.  If there is no good way to represent, we can remove it

Reply:
Now "feedback" has been removed from properties and required.
For example field, it's been replaced with "example_feedback"...Not super sure about this, but if we can elaborate more in documentation then this should work here.